mongo-init.js

```diff
@@ -0,0 +1,11 @@
+// Create Databases
+const dbPims = db.getSiblingDB('pims');
+const dbRemsAdmin = db.getSiblingDB('remsadmin');
+
+// Create Collections
+dbPims.createCollection('pimscollection');
+dbRemsAdmin.createCollection('remsadmincollection');
```

Comment:
Does this need to be an exhaustive list of all of the collections? Or can collections be added later by the applications?

Reply:
collections can be added later dynamically through the Mongo/Mongoose API.  Creating these collections doesn't really serve much as we will be creating different ones for use within the context of the code

Reply:
Should they be removed then?

Reply:
yeah I think so

Reply:
I removed the inserting of documents, but unless we create a collection or do something with the db it doesn't actually show up as a new data base within the mongo client after testing. I renamed the collections to be tmp collections which works for now and should be fine.